app/assets/javascripts/host_edit.js

```diff
@@ -146,13 +146,10 @@ function update_capabilities(capabilities) {
 var stop_pooling;
 
 function submit_with_all_params() {
-  var url = $('form').attr('action');
-  if (url.match('hostgroups')) {
-    resource = 'hostgroup';
-  } else {
-    resource = 'host';
-  }
-  capitalized_resource = resource[0].toUpperCase + resource.slice(1);
+  var url = $('form').attr('action'),
```

Comment:
I would prefer aligning the additional variable names at the same level (after `var` statement) in case we are keeping them.